google/cloud/firestore_admin_v1/services/firestore_admin/async_client.py

```diff
@@ -28,8 +28,8 @@
 from google.auth import credentials  # type: ignore
 from google.oauth2 import service_account  # type: ignore
 
-from google.api_core import operation as ga_operation  # type: ignore
-from google.api_core import operation_async  # type: ignore
+from google.api_core import operation as ga_operation
+from google.api_core import operation_async
```

Comment:
This is going to break `pytype` -- I can't tell why it isn't being run in https://source.cloud.google.com/results/invocations/02606fdf-7059-497e-931f-b5c4192081d1/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-firestore%2Fpresubmit%2Fpresubmit/log

The fix for it is in [this PR], released in [`gapic-generator-python 0.33.2`](https://github.com/googleapis/gapic-generator-python/releases/tag/v0.33.2).  Do we need to update the bazel config to use the newer version?